course_project/23/src/main/java/com/example/course_project/UsersController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.course_project;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping("users")
+public class UsersController {
+    private UsersRepository usersRepository;
```

Comment:
Typically you would want controllers to have reference to service classes, not repositories (because in service class you can do something else, like send email or send some notification)